app/assets/javascripts/checkbox.js

```diff
@@ -0,0 +1,22 @@
+$(document).on('ready turbolinks:load', function () {
```

Comment:
Yeah, I also wanted to get rid of the FILTER / SORT buttons with React and just let stuff happen automatically anytime a user click/change something. But the current implementation will be easier for the server to handle more traffic (way less requests with buttons).